src/components/cover/cover-image.tsx

```diff
@@ -26,7 +26,7 @@ const CoverImage: FC<CoverImageType> = ({
       { "cover__img--shadow": shadow }
     )}
     src={src}
-    alt={description || ""}
+    alt={altText}
```

Comment:
Consider what will happen if `altText` is not provided.

As I see it it was intentional that we used the empty string if no description was provided as [this marks the image as presentational](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attributes). I fear if we remove the `alt` attribute entirely if no value is provided. That may be intentional but I do not see anywhere else where we switch to passing an empty array.